lib/cutting_edge/app.rb

```diff
@@ -112,13 +114,14 @@ class App < Sinatra::Base
 
     get %r{/(.+)/(.+)/(.+)/svg} do |source, org, name|
       repo_defined?(source, org, name)
+      validate_token(params[:token]) if @repo.hidden?
```

Comment:
This requires exposing `CuttingEdge::SECRET_TOKEN` in the SVG link, no? I'm wondering if there a plausible use cases in which one might want to keep that token secret from collaborators on a private repo.

Reply:
Correct. Well, having the key would allow a collaborator from one private repo to see all the other private repos that are being monitored. That's why I suggested to maybe define a secret token per hidden repository.